de.dlr.sc.virsat.swtbot.test/src/de/dlr/sc/virsat/swtbot/test/ASwtBotTestCase.java

```diff
@@ -52,6 +55,19 @@
  */
 public class ASwtBotTestCase {
 
+	@Rule
+	public TestRule testStatusWatcher = new TestWatcher() {
+		@Override
+		protected void starting(Description description) {
+			System.err.println("SWTBotTestCase: " + description.getMethodName() + "...started");
```

Comment:
Any reason to output it in System.err instead of System.out?

Reply:
The basic idea was to see if it is possible to get direct out put into the TravisCI log rather then the redirected EclipseLog which is redirected into the file. Since it did not work i changed it to standard out now.